driver/driver_test.go

```diff
@@ -81,9 +81,16 @@ func TestDriverSuite(t *testing.T) {
 		account: &fakeAccountDriver{},
 		tags:    &fakeTagsDriver{},
 	}
-	defer driver.Stop()
 
-	go driver.Run()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	go func() {
+		err := driver.Run(ctx)
+		if err != nil {
```

Comment:
If we want to check this error reliably, we should probably call `cancel` after `sanity.Test` returns, and use a WaitGroup or channel to wait for this goroutine to shut down before the test ends. As it's written currently, I suspect the test will be finished before this line is executed, since the `defer` will execute right at the end of the test function.

Reply:
Ah, good catch.

I went with an `errgroup`. Let me know if that makes sense to you.

Reply:
Yep, makes sense to me :+1: